src/model.ts

```diff
@@ -914,6 +924,7 @@ export class GitExtension implements IGitExtension, IDisposable {
   private _readyPromise: Promise<void> = Promise.resolve();
   private _pendingReadyPromise = 0;
   private _poll: Poll;
+  private _refreshInterval = 3000; // ms
```

Comment:
If we remove `get refreshInterval` then it makes sense to also remove this private var, since it won't be used anywhere.

In general, in cases like this we should probably always try to work with values taken directly from the `settings` object, since that's effectively supposed to be the single source of truth.

Reply:
Removed.